AllCoreDevs-Meetings/Meeting 138.md

```diff
@@ -0,0 +1,802 @@
+
+**Andrew** ([1:19:25](https://youtu.be/BFxkGdPv4F8?t=4765))
+Yeah, its fine.
+
+**Tim** ([1:19:25](https://youtu.be/BFxkGdPv4F8?t=4765))
+Okay. Marek? Lukasz? Okay. And so, thank you Afr for the palindrome, , which I was too tired to recognize and so I’ll copy paste it here, also I’ll just share it in all core devs right now, so consider this the TTD value, I'll make it a proper upgrade to the Shanghai spec in the execution repo, and the thing that clients on the cl side need to figure out in the next week is basically the slot heights for, well basically the genesis for the beacon chain, and then the slot heights for Altair and Bellatrix based on this. Does that make sense?
+
+**Marius**
+Yes.
+
+**Tim** ([1:20:29](https://youtu.be/BFxkGdPv4F8?t=4829))
+Okay thank you Marius. Okay. So there's three people who wanted to discuss EIPs that are on the call. I doubt we can get through all of them, but if we stay on an additional five minutes, we can give them each five minutes, yeah so first up, Felix, you had EIP 44938. 
+
+**Felix**([1:21:03](https://youtu.be/BFxkGdPv4F8?t=4863))
+Yes. So and this can be really quick, so this isn't already like pre-agreed. I just wanted to let you guys know that for formal reasons, we are pursuing this EIP, the EIP 4938 is about removing the Geth Node data message from the Geth wire protocol, and we have discussed this extensively with all client teams that we are aware of, that we want to make this change in Geth, and we have been wanting to make this change in Geth for a very long time, and I can only really repeat what is in the EIP. We are set on making this change, because it will allow us to for example restructure our database to not store all of the tri nodes, for example by their hashes and we do provide an alternative to this protocol message in the snap protocol, and all of the existing users of Geth node data can be replaced by the messages in the snap protocol, so it is not but so can we just very quickly, can I just very quickly get from the client implementer some signal that this is okay.
```

Comment:
I think what I said here is, it **is** pre-agreed. Also, the EIP is about removing the **GetNodeData** message from the **eth** wire protocol.